src/utils/command_interface.py

```diff
@@ -159,7 +159,16 @@ def command_status(self, discord_id: str) -> str:
             if not challenge:
                 result += "No current challenges"
             else:
-                result += f"User {leetcode_user_id}'s Weekly Challenge status:\n"
+                questions = self.database.table_weeklyquestion_load_by_challenge_id()
```

Comment:
`table_weeklyquestion_load_by_challenge_id()` loads **by challenge id**. This should take an argument.